http_parser.gyp

```diff
@@ -47,6 +47,10 @@
     ],
   },
 
+  'variables': {
+    'http_max_header_size%': '8192'
```

Comment:
@bnoordhuis do you think we should change this back to 80KB here? We will be passing down the variable anyway in Node.js.

Reply:
I guess it won't hurt backwards compatibility but truthfully, I picked the 80 kb limit at random.